docs/articles/commands/module_lifespans.md

```diff
@@ -9,4 +9,47 @@ hand, are instantiated for each command call. This enables you to make use of tr
 unsure what that means, familiarize yourself with the [dependency injection](/articles/commands/dependency_injection.html "dependency injection") 
 guide.
 
-## TODO
\ No newline at end of file
+## 1. Default Implementation (Singleton)
+
+This is the default implementation where the ModuleLifespan Attribute does not need to be specified but can for completeness sake
+
+```cs
+using DSharpPlus.CommandsNext;
+using DSharpPlus.CommandsNext.Attributes;
+using System.Threading.Tasks;
+
+namespace MyFirstBot
+{
+    [ModuleLifespan(ModuleLifespan.Singleton)]
+    public class MyCommand : BaseCommandModule
+    {
+        [Command("hi")]
+        public async Task Hi(CommandContext ctx)
+        {
+            await ctx.RespondAsync($"👋 Hi, {ctx.User.Mention}!");
+        }
+    }
+}
+```
+
+## 2. Transient Implementation
+
+This is the implementation where the ModuleLifespan is set to be Transient
```

Comment:
While I believe it's explained elsewhere in the docs, it might be worth noting what it means that a module is transient. Specifically, unlike singleton, it is recreated every time it's used - advantages are that it allows for service scoping, among other things, however, unlike singleton modules, it does not persist state.

Reply:
It was explained in the opening paragraph I believe.  I will add it here though.